src/viam/api/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ if (VIAMCPPSDK_USE_DYNAMIC_PROTOS)
 endif()
 
 set(BUF_PROTO_COMPONENTS
+  app/mltraining
```

Comment:
Agree that this is now needed due to https://github.com/viamrobotics/api/pull/470